website/docs/operations.md

```diff
@@ -153,6 +153,29 @@ At a high level, this requires:
 * The ability to write to all objects in the group `mutations.gatekeeper.sh` (mutators)
 * The ability to read `MutatorPodStatus` objects in Gatekeeper's namespace
 
+## Mutation Controller
+
+__--operation key:__ `mutation-controller`
+
+This operation runs the process responsible for ingesting and registering
+mutators. `mutation-controller` is run implicitly with the `mutation-webhook`
+and `mutation-status` operations, and is redundant if any of the 2
+aforementioned operations are already specified. 
+
+If the `webhook` operation is used in isolation without the `mutation-webhook`
```

Comment:
```suggestion
If the `webhook` operation for validation is used in isolation without the `mutation-webhook`
```
Do we maybe want to also add this to workload-resources.md or a reference that links back to this section?

Reply:
How come audit doesnt require the mutation controller?

Reply:
Audit has `mutation-status`, which includes mutation controller

Reply:
> Audit has mutation-status, which includes mutation controller

ah that's right! forgot about that. might be good to document that audit already has `mutation-status` as a result of https://open-policy-agent.github.io/gatekeeper/website/docs/operations#mutation-status.